src/engine/framework/VirtualMachine.cpp

```diff
@@ -55,6 +59,26 @@ SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #define JOB_OBJECT_LIMIT_KILL_ON_JOB_CLOSE 0x2000
 #endif
 
+static Cvar::Cvar<bool> workaround_naclArchitecture_arm64_disableQualification(
```

Comment:
Should we ifdef these workarounds to their respective platforms?

Reply:
I was thinking about it but in the future I want to have a `dumCvars` command that would write some `dump.txt` file (basically doing [this](https://github.com/Unvanquished/unvanquished-documentation-data/blob/master/cvar-command-list/list) the non-hacky way and then I want all cvars to be listed…